config/checkstyle_checks.xml

```diff
@@ -557,6 +557,12 @@
       <property name="file" value="${checkstyle.importcontroltest.file}"/>
       <property name="path" value="^.*[\\/]src[\\/]test[\\/].*$"/>
```

Comment:
@romani I noticed nothing is maintaining "it" folder via import control. I don't see why test should not cover it since it is all tests anyways.

Reply:
@shashwatj07 , it is good point, lets cover it also, all violations goes to suppression.

Reply:
moved to https://github.com/checkstyle/checkstyle/issues/10074
please approve, I'll send PR soon

Reply:
It was approved.

Reply:
PR opened